net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -2641,7 +2647,29 @@ void RewriteOptions::InitOptionNameToPropertyArray() {
   }
 }
 
+void RewriteOptions::InitFixedResourceHeaders() {
+  int index = 0;
+  StringPieceVector* tmp = new StringPieceVector(12);
```

Comment:
i probably wouldn't bother with estimating the size here; just say 'new StringPieceVector;', and use push_back below.  This will only be run once on process startup.